terraform/main.tf

```diff
@@ -50,7 +50,7 @@ module "batch_gdal_python_image" {
   image_name      = substr(lower("${local.project}-gdal_python${local.name_suffix}"), 0, 64)
   root_dir        = "${path.root}/../"
   docker_path     = "batch"
-  docker_filename = "gdal-python.dockerfile"
+  docker_filename = "universal_batch.dockerfile"
```

Comment:
**Can you have a single 'module "universal_batch_image" at line 48, adjust the image_name at line 50, remove lines 65-81, and then adjust lines 224, 226,227 to use the same docker url based on the common image name?  That way, you're only creating one docker instead of 3 during each data API deployment?

Reply:
Good point, will combine, thanks!